.travis.yml

```diff
@@ -4,9 +4,12 @@ go:
   - "1.10.x"
   - master
 
+go_import_path: github.com/driusan/dgit
+
 script:
+    - pwd
     - diff -u <(echo -n) <(gofmt -d ./)
     - go test -v ./...
     - chmod u+x run-official-tests.sh
-    - ./run-official-tests.sh t0000-basic.sh
+    - ./run-official-tests.sh -i t0000-basic.sh t0011-hashmap.sh t0040-parse-options.sh t0062-revision-walking.sh t0063-string-list.sh t0064-sha1-array.sh t0070-fundamental.sh t0081-line-buffer.sh t0110-urlmatch-normalization.sh t0200-gettext-basic.sh
```

Comment:
t0011-hashmap.sh, t0040-parse-options.sh, t0063-string-list.sh, t0064-sha1-array.sh, t0110-urlmatch-normalization.sh and t0200-gettext-basic.sh don't seem to ever invoke dgit. I think this is mostly testing test-tool, which is built by git.

I think the most important ones (which are currently failing, but we currently allow it to fail) are the -m and -u read-tree tests. Maybe some of the update-index ones. Basically, things that can corrupt the index. (I'd say write-tree was even more important, but I don't see any tests specifically testing write-tree.)

Reply:
I was adding the suites that seem to be passing fine in the hopes of eventually just running all of them. I'll revert back to the ones that we think are actually testing dgit.

Reply:
We could consider writing a test-tool replacement which tests *our* internals in the same way and hiding it behind a build flag (I haven't looked into how complicated that would be), but until running the tests fails the build I don't see much point in running the tests which aren't testing our functionality.